packages/select/src/components/multi-select/multiSelect.tsx

```diff
@@ -221,6 +239,13 @@ export class MultiSelect<T> extends AbstractPureComponent<MultiSelectProps<T>, M
                 className={classNames(listProps.className, popoverProps.className)}
                 content={
                     <div {...popoverContentProps} onKeyDown={handleKeyDown} onKeyUp={handleKeyUp}>
+                        {/* If customTarget is provided, TagInput is not rendered. This loses the search input.
+                        To account for this, render the search input within the popover similar to Select.
+
+                        Clearing all items is still possible since this component is controlled. It just not cannot
+                        be through the default button in this component, rather done through a custom button that can be
+                        rendered from within the popover through the itemListRenderer or from externally. */}
+                        {this.props.customTarget != null && input}
```

Comment:
highlighting this as a TODO - let's catch up on this once everything else is looking good